fluentd/lib/fluent-plugin-viaq_data_model/lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -316,37 +335,27 @@ def process_sys_var_log_fields(tag, time, record, fmtr = nil)
         end
         record['time'] = timeobj.utc.to_datetime.rfc3339(6)
       end
-      if record['host'].eql?('localhost') && @docker_hostname
-        record['hostname'] = @docker_hostname
+      if record['host'].eql?('localhost') && @node_name
+        record['hostname'] = @node_name
       else
         record['hostname'] = record['host']
       end
     end
 
     def process_k8s_json_file_fields(tag, time, record, fmtr = nil)
-      record['message'] = record['message'] || record['log']
+      # TODO remove this line once parser changes merge. assume 'message' is the default
+      record['message'] = record['log'] if record['message'].nil?
       normalize_level!(record)
-      if record.key?('kubernetes') && record['kubernetes'].respond_to?(:fetch) && \
-         (k8shost = record['kubernetes'].fetch('host', nil))
+      if record.key?('kubernetes') && record['kubernetes'].respond_to?(:key?) && \
+         (k8shost = record['kubernetes']['host'])
```

Comment:
This will work for Hash objects, but since you're calling [] it would be better to use `respond_to :[]`.
```suggestion
      if record.key?('kubernetes') && record['kubernetes'].respond_to?(:[]) && \
         (k8shost = record['kubernetes']['host'])
```